.devcontainer/Dockerfile

```diff
@@ -1,32 +1,13 @@
-# FROM mcr.microsoft.com/vscode/devcontainers/rust:latest
-FROM ghcr.io/plc-lang/rust-llvm:latest-linux
+FROM mcr.microsoft.com/vscode/devcontainers/rust:latest
```

Comment:
Count we move this change to another commit / PR?
I had the other version working using docker on another PC. And I prefer we look more into it

Reply:
> This still segfaults for me. I ran it through the debugger and this is the stack trace. So I think we can accept this change for the string fixes but we cannot close the issue yet
> 
> ![image](https://user-images.githubusercontent.com/35030/149712844-d3132739-f86b-43fd-8945-25ea8299b355.png)

https://llvm.org/doxygen/ConstantHoisting_8h_source.html This is the pass

This is the link to the method that is panicking
https://llvm.org/doxygen/Local_8cpp_source.html#l03248

Reply:
> Count we move this change to another commit / PR? I had the other version working using docker on another PC. And I prefer we look more into it

I removed the files, I accidentally added them